plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraPageSink.java

```diff
@@ -97,23 +92,19 @@ public CassandraPageSink(
         this.generateUuid = generateUuid;
         this.batchSize = batchSize;
 
-        if (protocolVersion.toInt() <= ProtocolVersion.V3.toInt()) {
```

Comment:
Why do you want to remove support for protocol version 3 in addition to 2? Relates to https://github.com/trinodb/trino/issues/7729#issuecomment-826836017?

Reply:
Good catch. I missed this aspect even though I properly modified `io.trino.plugin.cassandra.CassandraType#toCassandraType(io.trino.spi.type.Type, com.datastax.oss.driver.api.core.ProtocolVersion)`
I added corresponding tests to cover this use case as well.